docs/src/main/asciidoc/_migration-guide-for-4.0.adoc

```diff
@@ -403,11 +403,11 @@ Changes for the child entries for following prefix, please refer the following t
 [cols="<30,<~", options="header"]
 |===
 |Legacy properties |Modern properties
-|consumer.max-batch-size |consumer.batch.max-size
-|consumer.max-wait-time |consumer.batch.max-wait-time
-|consumer.checkpoint-mode |consumer.checkpoint.mode
-|consumer.checkpoint-count |consumer.checkpoint.count
-|consumer.checkpoint-interval |consumer.checkpoint.interval
+|*consumer*.max-batch-size |*consumer*.batch.max-size
```

Comment:
How about add `*` for the strings that appears in `@ConfigurationProperties("...")`?